sdk/appcenter-auth/src/main/java/com/microsoft/appcenter/auth/Constants.java

```diff
@@ -86,4 +91,39 @@
      * JSON configuration key for identity scope.
      */
     static final String IDENTITY_SCOPE = "identity_scope";
+
+    /**
+     * JSON configuration key for audience within {@link #AUTHORITIES} array.
+     */
+    static final String Audience = "audience";
```

Comment:
All constants need to be UPPER_UNDERSCORE_CASING

Reply:
Got it. Changed all constants to upper case.

Reply:
I think you missed Audience itself.

Reply:
Thanks Michael